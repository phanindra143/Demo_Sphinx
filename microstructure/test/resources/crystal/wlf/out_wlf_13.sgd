*if,ifprop,eq,2,then
  !c_ij(1,1)=     0.0000
  !c_ij(1,2)=     0.0000
  !c_ij(2,2)=     0.0000
  !c_ij(1,3)=     0.0000
  !c_ij(2,3)=     0.0000
  !c_ij(3,3)=     0.0000
  !c_ij(1,4)=     0.0000
  !c_ij(2,4)=     0.0000
  !c_ij(3,4)=     0.0000
  !c_ij(4,4)=     0.0000
  !c_ij(1,5)=     0.0000
  !c_ij(2,5)=     0.0000
  !c_ij(3,5)=     0.0000
  !c_ij(4,5)=     0.0000
  !c_ij(5,5)=     0.0000
  !c_ij(1,6)=     0.0000
  !c_ij(2,6)=     0.0000
  !c_ij(3,6)=     0.0000
  !c_ij(4,6)=     0.0000
  !c_ij(5,6)=     0.0000
  !c_ij(6,6)=     0.0000
*endif
*if,ifprop,eq,1,then
  !k_ij(1,1)=   113.7840
  !k_ij(1,2)=     0.5528
  !k_ij(2,2)=   114.1112
  !k_ij(1,3)=    -0.3005
  !k_ij(2,3)=     0.5099
  !k_ij(3,3)=   113.6557
MP,Kxx,1,  113.78398
MP,Kyy,1,  114.11123
MP,Kzz,1,  113.65565
MP,Kxx,2,  114.11123
MP,Kyy,2,  113.78398
MP,Kzz,2,  113.65565
MP,Kxx,3,  114.11123
MP,Kyy,3,  113.78398
MP,Kzz,3,  113.65565
MP,Kxx,4,  113.78398
MP,Kyy,4,  114.11123
MP,Kzz,4,  113.65565
MP,Kxx,5,  113.78398
MP,Kyy,5,  114.11123
MP,Kzz,5,  113.65565
MP,Kxx,6,  114.11123
MP,Kyy,6,  113.78398
MP,Kzz,6,  113.65565
MP,Kxx,7,  114.11123
MP,Kyy,7,  113.78398
MP,Kzz,7,  113.65565
MP,Kxx,8,  113.78398
MP,Kyy,8,  114.11123
MP,Kzz,8,  113.65565
MP,Kxx,9,  113.65565
MP,Kyy,9,  113.78398
MP,Kzz,9,  114.11123
MP,Kxx,10,  113.78398
MP,Kyy,10,  113.65565
MP,Kzz,10,  114.11123
MP,Kxx,11,  113.78398
MP,Kyy,11,  113.65565
MP,Kzz,11,  114.11123
MP,Kxx,12,  113.65565
MP,Kyy,12,  113.78398
MP,Kzz,12,  114.11123
MP,Kxx,13,  113.65565
MP,Kyy,13,  113.78398
MP,Kzz,13,  114.11123
MP,Kxx,14,  113.78398
MP,Kyy,14,  113.65565
MP,Kzz,14,  114.11123
MP,Kxx,15,  113.78398
MP,Kyy,15,  113.65565
MP,Kzz,15,  114.11123
MP,Kxx,16,  113.65565
MP,Kyy,16,  113.78398
MP,Kzz,16,  114.11123
MP,Kxx,17,  114.11123
MP,Kyy,17,  113.65565
MP,Kzz,17,  113.78398
MP,Kxx,18,  113.65565
MP,Kyy,18,  114.11123
MP,Kzz,18,  113.78398
MP,Kxx,19,  113.65565
MP,Kyy,19,  114.11123
MP,Kzz,19,  113.78398
MP,Kxx,20,  114.11123
MP,Kyy,20,  113.65565
MP,Kzz,20,  113.78398
MP,Kxx,21,  114.11123
MP,Kyy,21,  113.65565
MP,Kzz,21,  113.78398
MP,Kxx,22,  113.65565
MP,Kyy,22,  114.11123
MP,Kzz,22,  113.78398
MP,Kxx,23,  113.65565
MP,Kyy,23,  114.11123
MP,Kzz,23,  113.78398
MP,Kxx,24,  114.11123
MP,Kyy,24,  113.65565
MP,Kzz,24,  113.78398
*endif
