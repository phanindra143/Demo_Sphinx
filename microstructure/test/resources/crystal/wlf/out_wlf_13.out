            ifx                 q_x                 q_y                 q_z     
  /% source: R:/166_320/Ablage_Simulationen/Modellbildung_Simulation_CMC-SiC_[Vorlauf+WiMi+MaVo]/04_04_2017_Modellierung_Matrix_mikro_ditt_[WiMi]/2_Waermeleitfähigket_SiC-Si\WLF-50SiC-Poren\5
             3.00000      -1777857.10207         -4970.04829          5399.93392
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             3.00000         15625.00000           -33.41974            -4.17487
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             2.00000         -1096.43152      -1782895.56220          4155.52865
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             2.00000           -65.26105         15625.00000          -108.76381
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             1.00000          6186.51423          2042.11762      -1775831.67292
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             1.00000           -14.57149           -85.72343         15625.00000
  /%
